subsys/net/lib/nrf70_fw_ext/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if (not CONFIG_NRF_WIFI_PATCHES_BUILTIN)
+zephyr_library_sources_ifdef(ext_fw_load.c)
+if(CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP)
+# Run patches from the external flash (XIP). No need to copy.
+zephyr_code_relocate(FILES ext_fw_load.c LOCATION EXTFLASH_RODATA NOCOPY)
+endif()
```

Comment:
this cannot have been tested because it is missing an endif

Reply:
I did only test build for XIP, this might have creptin with some rebase later.